Foundation/NSTask.swift

```diff
@@ -7,6 +7,7 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
+#if !os(Android) // not available
```

Comment:
Yeah, that's a good point. I'm not 100% sure -- I just grepped around, found that spot, and thought it looked interesting. :)

If adding Android to that list is indeed what @parkera had in mind, we could email the swift-dev mailing list to learn more about it.